crates/ruff_python_formatter/src/other/parameters.rs

```diff
@@ -246,13 +246,16 @@ impl FormatNodeRule<Parameters> for FormatParameters {
             // No parameters, format any dangling comments between `()`
             write!(
                 f,
-                [empty_parenthesized("(", ")").with_dangling_comments(dangling)]
+                [empty_parenthesized("(", ")")
+                    .with_dangling_comments(dangling)
+                    .with_ungroup()]
```

Comment:
E.g., here is where we're ungrouping.

Reply:
Is this ungrouping necessary for the empty parenthesized case? I believe the extra group is unnecessary because a trailing comment will always make all parent groups extend. In addition, using a group for empty parenthesized may always be incorrect because we never want the printer to break right after the `(` 

```python
a = bbbbbbbbbbbbbbbbbbbbb + ()
                                                             ^ max line width

# We don't want
a = bbbbbbbbbbbbbbbbbbbbb + (
)
```



```rust

let (end_of_line_comments, own_line_comments) = self.comments.split_at(end_of_line_split);

write!(
	f, 
	[
		text(self.left),
		trailing_comments(end_of_line_comments),
	]
)?;

if !own_line_comments.is_empty() {
	block_indent(&dangling_comments(own_line_comments)).fmt(f)?;
} else if !end_of_line_comments.is_empty() {
	hard_line_break.fmt(f)?;
}
		
text(self.right).fmt(f)
```